qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -283,6 +286,18 @@ HOCRPdfExporter::HOCRPdfExporter(const HOCRDocument* hocrdocument, const HOCRPag
 	ui.comboBoxImageCompression->addItem(_("Jpeg (lossy)"), PDFSettings::CompressJpeg);
 	ui.comboBoxImageCompression->setCurrentIndex(-1);
 
+	ui.comboBoxPaperSizeMetric->addItem(_("mm"), static_cast<int>(PaperSizeMetrics::mm));
+	ui.comboBoxPaperSizeMetric->addItem(_("inch"), static_cast<int>(PaperSizeMetrics::inch));
+
+	ui.comboBoxPaperSize->addItem("Same as source", "Same as source");
+	ui.comboBoxPaperSize->addItem("Custom", "Custom");
```

Comment:
Label needs to localizable. Usually you don't use human strings as data.

Reply:
Fixed.